linera-views/src/common.rs

```diff
@@ -171,23 +171,23 @@ where
         }
     }
 
-    pub(crate) fn is_index_absent(&mut self, index: &[u8]) -> bool {
+    pub(crate) fn contains_key(&mut self, index: &[u8]) -> bool {
```

Comment:
No, we cannot code it to be `&self` because there is an increase in the iterators.

Reply:
I wrote `self` (or `mut self` in this case)

Reply:
But then we would consume the iterator and we could not call it again. So, I do not see how that could be correct.

Reply:
Ok, I see that we are calling this function repeatedly indeed. Because it is mutating the iterator, we cannot call it `contains_key` in this case. Can you rename it into `find_key` ?

Reply:
Of course, I can.